django-rgd-imagery/rgd_imagery/serializers/stac/item/bands.py

```diff
@@ -75,6 +75,7 @@ def to_model(eo_band: Band, image: models.Image):
             Decimal(eo_band.center_wavelength) + Decimal(eo_band.full_width_half_max) / 2
         )
         eo_band_spectral_lower = eo_band_spectral_upper - Decimal(eo_band.full_width_half_max) / 2
+    # TODO: 'eo_band_spectral_lower' referenced before assignment
     bandmeta.band_range = (
         eo_band_spectral_lower,
         eo_band_spectral_upper,
```

Comment:
@mcovalt, thoughts on how to resolve this?

Reply:
The serializer is still trying to save spectral information, but none was provided. This ought to be fixed with https://github.com/ResonantGeoData/ResonantGeoData/pull/549/commits/fcab4ce4cbfc4a1ee511d62c4b98eaaa0998ab8e.